test/test_validation.py

```diff
@@ -196,3 +196,31 @@ def test_prop_string_values(self):
                               values=['(39.12; 89; 67.19)', '(39.12; 78; 67.19)', '(39.12)'])
         self.assertError(validate(prop9),
                          'Dtype of property "Coos" currently is "string", but might fit dtype "3-tuple"!')
+
+    def test_load_xml(self):
```

Comment:
Please split this test into individual tests for a) undefined section type and b) properties with undefined dtypes. Same for the `json` and `yaml` tests.